projects/driver_controls/inc/throttle.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+// Module for the throttle.
+// Requires Ads1015, and soft timer to be initialized.
+// To use the module, init the ADS1015 for the pedal and pass its Ads1015Storage to throttle_init,
+// along with the two channels the pedal is connected to. At any time calling throttle_get_position
+// will give the current position of the pedal.
+// Note that throttle_storage should persist across functions.
+#include <stdint.h>
+#include "ads1015.h"
+#include "soft_timer.h"
+#include "status.h"
+
+typedef enum {
+  THROTTLE_ZONE_BRAKE = 0,
+  THROTTLE_ZONE_COAST,
+  THROTTLE_ZONE_ACCEL,
+  NUM_THROTTLE_ZONES
+} ThrottleZone;
+
+// A measure in a 12 bit scale of how far within a zone a pedal is pressed.
+typedef uint16_t ThrottleFraction;
+
+typedef struct ThrottlePosition {
+  ThrottleZone zone;
+  ThrottleFraction fraction;
+} ThrottlePosition;
+
+typedef struct ThrottleCalibrationData {
+  int16_t main_bottom_thresh;
```

Comment:
What do each of these represent? Shouldn't there be min/max bounds?

Reply:
bottom is min and accel is max. Adding a comment is sufficient or there is a better naming scheme?

Reply:
Why is accel max? That naming scheme seems odd.